src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -193,12 +193,37 @@ public static void onPlayerDestroyItem(Player player, @NotNull ItemStack stack,
      * Vanilla calls to {@link Mob#finalizeSpawn} are replaced with calls to this method via coremod.<br>
      * Mods should call this method in place of calling {@link Mob#finalizeSpawn}. Super calls (from within overrides) should not be wrapped.
      * <p>
-     * Returns the SpawnGroupData from this event, or null if it was canceled.
+     * When interfacing with this event, write all code as normal, and replace {@link Mob#finalizeSpawn} with {@link #onFinalizeSpawn}.<p>
```

Comment:
Instead of a link to this method, it should just refer to "this method":
```suggestion
     * When interfacing with this event, write all code as normal, and replace the call to {@link Mob#finalizeSpawn} with a call to this method.<p>
```